samples/sqlservices/typings/sqlops.proposed.d.ts

```diff
@@ -0,0 +1,203 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// This is the place for API experiments and proposal.
+
+import * as core from 'sqlops';
```

Comment:
do we really need to duplicate this file in the sample?  won't it be difficult to keep up to date?

Reply:
I can remove and use as relative. Good suggestion